BB/ActiveTimedTasks.py

```diff
@@ -3,4 +3,6 @@
 shopRefreshTT = None
 dbSaveTT = None
 
-duelRequestTTDB = None
```

Comment:
ah yes, this line was changed as the stylistic warning was pushed to a different line